library/alloc/src/collections/btree/node/tests.rs

```diff
@@ -114,6 +114,5 @@ fn test_partial_cmp_eq() {
 fn test_sizes() {
     assert_eq!(core::mem::size_of::<LeafNode<(), ()>>(), 16);
     assert_eq!(core::mem::size_of::<LeafNode<i64, i64>>(), 16 + CAPACITY * 8 * 2);
-    assert_eq!(core::mem::size_of::<InternalNode<(), ()>>(), 112);
-    assert_eq!(core::mem::size_of::<InternalNode<i64, i64>>(), 112 + CAPACITY * 8 * 2);
+    assert_eq!(core::mem::size_of::<InternalNode<(), ()>>(), 16 + (CAPACITY + 1) * 8);
```

Comment:
Why did we remove the additional assert here?

Reply:
It seems to have very little chance of detecting anything missed by the 2nd and 3rd assert, for a complicated formula, but I'll add it again.